### PR TITLE
feat: adding active directory integration

### PR DESCRIPTION
**Pull request by thezackm**
# Summary

Adding quickstart, data source, alerts, and dashboard for new Active Directory integration. 

> ~‼️  Pending live URL from the [docs site](https://github.com/newrelic/docs-website/pull/13244)~ 🎉  Completed!

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/active-directory/active-directory-windows-services.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Active Directory Windows Services
+description: |+
+  This alert is triggered when the any of the targeted Windows Services are in a state other than "running".
```
Comment (seemantk) [inline alert-policies/active-directory/active-directory-windows-services.yml:3]:
```suggestion
  This alert is triggered when any of the targeted Windows Services are in a state other than "running".
```

File: quickstarts/active-directory/config.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+slug: active-directory
+title: Active Directory
+description: |
+  This quickstart provides you with the integration, dashboard, and alert policy necessary to proactively monitor the health and availability of your Active Directory domain controllers.
```
Comment (seemantk) [inline quickstarts/active-directory/config.yml:4]:
```suggestion
  This quickstart provides you with the integration, dashboard, and alert policies necessary to proactively monitor the health and availability of your Active Directory domain controllers.
```

**Comment by thezackm** [comment]
@seemantk - suggestions applied; thanks!

**Review by seemantk — APPROVED** [review]
Thank you @thezackm this LGTM

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**